src/lib/apiTypesV2/HttpInfo.h

```diff
@@ -47,6 +47,7 @@ struct HttpInfo
   std::map<std::string, std::string>  headers;
   std::string                         payload;
   bool                                custom;
+  bool                                includePayload;
```

Comment:
Class constructors in HttpInfo.cpp should be modified to ensure that includePayload is initialized with the right default value (true).

Reply:
Did you finally modify the constructor as described in comment above, please? Maybe I have missed it but I don't see the change in the PR...